plugins/source/azure/client/multiplexer.go

```diff
@@ -1,24 +1,14 @@
 package client
 
-import "github.com/cloudquery/plugin-sdk/schema"
-
-// SingleSubscriptionMultiplex can be used in cases where any single subscription would do. This can reduce the number
-// of API calls.
-func SingleSubscriptionMultiplex(meta schema.ClientMeta) []schema.ClientMeta {
```

Comment:
What's the reason this was removed? I remember we needed this in https://github.com/cloudquery/cloudquery/pull/2018

Reply:
1 - I'm adding new resources now, will publish the PR once all resources are ready
2 - I'll look into that
3 - https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/services@v67.1.0+incompatible (almost every package under this path is being decommissioned in favor of the new mgmt approach)